version/constants.go

```diff
@@ -15,15 +15,15 @@ const (
 	// RPCChainVMProtocol should be bumped anytime changes are made which
 	// require the plugin vm to upgrade to latest avalanchego release to be
 	// compatible.
-	RPCChainVMProtocol uint = 41
+	RPCChainVMProtocol uint = 42
```

Comment:
When we change the protocol version we also update the buf readme (`proto/README.md`) accordingly.